client.go

```diff
@@ -192,6 +199,10 @@ func (c *Client) exchange(m *Msg, a string) (r *Msg, rtt time.Duration, err erro
 	// Otherwise use the client's configured UDP size.
 	if opt == nil && c.UDPSize >= MinMsgSize {
 		co.UDPSize = c.UDPSize
+		// In this case also set the the EDNS0 bufsize to allow for larger responses.
+		if c.SetUDPSize {
+			m.SetEdns0(c.UDPSize, true /* dnssec */)
```

Comment:
Yeah, I'm wondering about this as well. Maybe SetUDPSize should be an int:
* 0: off
* 1: enable, without DNSSEC
* 2: enable, with DNSSEC

?